.vscode/extensions.json

```diff
@@ -0,0 +1,24 @@
+{
+  "recommendations": [
```

Comment:
@bdittes introduced a number of extensions suggestions for vscode that I find interesting. I've installed most of them (except for the git ones) and are good tools.
Being suggestions imply they will be recommended by the editor but you don't need to install them mandatorily. 

What you guys think? @RafaPolit @konzz @daneryl @elric-wamugu @gabriel-piles

Reply:
@fnocetti what about not actually include specific ide/editor related configs on the project? is this a common practice ? can the ppl interested maintain a separate repo ? im ok if its merged here just to check what the others think

Reply:
@daneryl that was my first thought but since we are currently maintaining a .vscode folder with configs I wanted to know what you folks think...